### ISSUE TITLE
OutOfRangeError (see above for traceback): End of sequence 

### ISSUE THREAD

**Issue by Alommo** [aloyschen/tensorflow-yolo3#6]
Hi, I recieved the error when running 'yolo_train.py'. Do you have a similar situation？Thanks very much!

**Comment by aloyschen** [comment]
Do you create the tfrecords successfully？

**Comment by Alommo** [comment]
Thanks for your reply! I think it's a bug . When I'm using tensorflow1.6 instead of tensorflow1.11 or 1.8 i successed.

**Comment by jjwwczy** [comment]
I met this problem too. But when I change the batchsize from 2 to 16, it is solved.
我也见过这个问题，但是是把batchsize从2改成16就好了

**Comment by ch135** [comment]
I also have this problem while I'm using tehsorflow1.12. It still exits when the train_batch_size and val_batch_size are 16. Could you help me?

**Comment by jjwwczy** [comment]
I am not sure, maybe it's about your memory size of GPU. Try to lower the train_batch_size?

**Comment by ch135** [comment]
Thank you, I have solven this problem because of the loss of threcords and I have low the train_batch_size as you said. 
While, I have another preblem. When I am training, the size of computer memory( 8g ) outs using tensorflow-gpu. I am very confusion of it. Why isn't the GPU memory out ?

**Comment by jjwwczy** [comment]
I didn't meet such problem. I guess you should check if you have installed the cuda and cudnn correctly.
BTW, if you can speak Chinese, it is more comprehensive for me.

**Comment by ch135** [comment]
谢谢，我已经用tensorflow-gpu 跑过其他模型了，但在 yolo 训练时，电脑内存爆满，gpu没使用到。我想gpu加载资源时，是不是先加载到电脑内存中，再由 cpu 控制传送到 gpu 中，这样就会出现这种情况，毕竟 tfrecodes 很大。

**Comment by jjwwczy** [comment]
这个我也不清楚，加载数据之后你能正常训练吗？ 我发现数据预处理很慢，所以把DataReader的第315行的shuffle改小了一点，后面就比较快了。
`     dataset = dataset.repeat().shuffle(300).batch(batch_size).prefetch(batch_size)`

**Comment by ch135** [comment]
多谢，我按你说的改完后现在可以跑了。我想请教下这句话是什么意思呢？改完后电脑内存刚好可以用

**Comment by jjwwczy** [comment]
打乱数据用的，是tensorflow自带的api

**Comment by ch135** [comment]
2018-12-04 19:38:48.091818: W tensorflow/core/framework/op_kernel.cc:1273] OP_REQUIRES failed at save_restore_v2_ops.cc:109 : Not found: Failed to create a NewWriteableFile: ./test_model/model.ckpt-192192\model.ckpt-11828.data-00000-of-00001.tempstate8019304214475019900 : ϵͳ�Ҳ���ָ����·����
; No such process

请问这个项目的test_model.ckt-192192 在哪里下载的呢

**Comment by jjwwczy** [comment]
你初次训练要改一下config.py，model_dir的位置。 如果是文件夹，那么会在文件夹下生成训练的模型。如果是模型，那么会寻找checkpoint在原模型的基础上继续训练。

**Comment by ch135** [comment]
嗯，我直接改成 model_dir = './test_model' 了
